legal_references.tsv

```diff
@@ -799,7 +798,8 @@ lov om personell i forsvaret|forsvarspersonelloven	lov/2004-07-02-59
 lov om personnavn|navneloven	lov/2002-06-07-19
 lov om personopplysninger|personopplysningsloven|pol	lov/2000-04-14-31
 lov om petroleumsvirksomhet|petroleumsvirksomhetsloven|petroleumsloven|petrl	lov/1996-11-29-72
-lov om planlegging og byggesaksbehandling|plan- og bygningsloven|bygningsloven|bygningsloven|plbl|pbl	lov/2008-06-27-71
+lov om planlegging og byggesaksbehandling	lov/2008-06-27-71
+plan- og bygningsloven|bygningslov|bygningsloven|pbl	lov/1985-06-14-77|lov/1965-06-18-7|lov/1924-02-22-2
```

Comment:
`lov/2008-06-27-71` kan jo også refereres til som "plan- og bygningsloven" eller "pbl". Her burde vi være litt mer nøye med hvordan vi splitter opp.

Reply:
Rart, spurte Rolf om det. Tror det er bedre at jeg sender Rolf hele fila slik at han kan jobbe med den selv. Det er jo ikke veldig komplisert filstruktur uansett.

Reply:
Kan være greit om han kan redigere den, men det er en del subtleties med hvordan denne fila tolkes som burde tydeliggjøres i så fall:

* Vi søker igjennom den aktuelle teksten etter referansene nevnt i denne fila. Om flere linjer (aka regexer) matcher, hvilken får presedens? Med andre ord, har rekkefølgen på linjene noe å si?
* Om matchende regex mapper til flere forskjellige dokumenter, hvilke av disse får presedens? Om et årstall nevnes etter dokumentnavnet i teksten, så er det vel dokumentet med det årstallet som får presedens, ellers blir det nyeste, sant?
* Om man har et mange-til-mange forhold mellom navn og dokumenter, hvordan skal de splittes opp over linjer? Som et forenklet eksempel, om `lov/2008-06-27-71` kan refereres til med "Lov om planlegging og byggesaksbehandling", "plan- og bygningsloven" eller "pbl", og `lov/1985-06-14-77` kan refereres til med "plan- og bygningslov", "plan- og bygningsloven" eller "plbl", hvordan bør disse fordeles over linjer? Det viktige her er vel at vi grupperer på hvilke dokumenter et navn kan referere til. Så dette kunne vært skrevet som

    ```
    lov om planlegging og byggesaksbehandling	lov/2008-06-27-71
    pbl	lov/2008-06-27-71
    plan- og bygningsloven	lov/2008-06-27-71|lov/1985-06-14-77
    plan- og bygningslov	lov/1985-06-14-77
    plbl	lov/1985-06-14-77
    ```

    som kunne forenkles til å slå sammen linjer som referere til samme sett med dokumenter, slik:

    ```
    lov om planlegging og byggesaksbehandling|pbl	lov/2008-06-27-71
    plan- og bygningsloven	lov/2008-06-27-71|lov/1985-06-14-77
    plan- og bygningslov|plbl	lov/1985-06-14-77
    ```
* Burde relaterte linjer (som i eksempelet over) grupperes på et vis? I så fall hvordan? Eller er det helt likegyldig hvor i TSV-fila de ligger? Om linjene sorteres alfabetisk vil de jo ikke være gruppert, men grupperinga kan være ganske nyttig siden dette er en manuelt vedlikeholdt fil.
* Sørg for å bruke tab istedet for mellomrom for å skille mellom navneseksjonen og lov-ID-seksjonen.
* Sørg for å bruke `|` for å skille mellom navn på samme linje, og for å skille mellom lov-IDer på samme linje.
* Har rekkefølgen av navnene på en linje noe å si?
* Har rekkefølgen av lov-IDene på en linje noe å si?
* Har store og små bokstaver noe å si? Er det forskjell på å skrive "Plan- og bygningsloven" og "plan- og bygningsloven"?
* Har det noe å si hvilken form man skriver navnet i? F.eks. ubestemt form "arbeidsmiljølov" kontra bestemt form "arbeidsmiljøloven", og evt. rekkefølgen på disse? Antagelig er det best at de lengste formene er først, sånn at vi ikke lenker `<Link>Arbeidsmiljølov</Link>en`.

Kort fortalt, denne fila virker kanskje triviell, men det er veldig mye å ta hensyn til som påvirker hvilke lenker som til syvende og sist foreslås. Om redaksjonen skal redigere denne fila direkte (eller egentlig uansett, dette vil være nyttig for oss også) så vil det være nyttig med en README på hvordan denne fila tolkes, som forklarer alle punktene over.